lib/internal/bootstrap/context.js

```diff
@@ -19,26 +19,18 @@ const ReflectApply = global.Reflect.apply;
 const warning = 'Atomics.wake will be removed in a future version, ' +
   'use Atomics.notify instead.';
 
-let wakeWarned = false;
 function wake(typedArray, index, count) {
-  if (!wakeWarned) {
-    wakeWarned = true;
-
-    if (global.process !== undefined) {
-      global.process.emitWarning(warning, 'Atomics');
-    } else {
-      global.console.error(`Atomics: ${warning}`);
-    }
-  }
-
+  global.console.error(`Atomics: ${warning}`);
```

Comment:
we really should use emitWarning here. the only reason the alternative exists is to warn usage of wake within node core itself before process is set up.

Reply:
The proper warning is created afterwards in line 359 of bootstrap.js where it gets a deprecation code as well.

Reply:
that won't work. context.js is run for every context we create, but node.js only runs for the main context

Reply:
Is there any other way to access the native `require` in context.js?

Reply:
Or can `process` be passed as an argument through the context.js wrapper?

Reply:
process doesn't exist yet, that's why it's checked lazily via global.process

Reply:
For now I'm going ahead with the docs-only deprecation, where the global / process context is created but the global accesses won't throw deprecation warnings, so I'm leaving this file as-is for now.